WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -140,6 +140,24 @@ final class SiteAddressViewController: LoginViewController {
         }
     }
 
+    override func displayRemoteError(_ error: Error) {
```

Comment:
This might not be necessary. In fact, I would argue that it would be good to avoid doing it.

Reply:
I'm sorry, but I'm confused by this. It appears you added this override method to implement your error handling. Why do you not agree with what you just did? 😄

Reply:
Sorry, that was terrible wording on my side. Let me rephrase my comment:

"I am not 100% sure that this is absolutely necessary. So far I have not seen this override being used in any of the tests I have run using WooCommerce. That does not mean it won't happen as I continue testing. I would prefer if, eventually, we noticed that overriding this method is actually unnecessary and we could get rid of the override"

Reply:
And I just found a test case for WooCommerce where these overrides are necessary: a Pressable store with Jetpack installed but not activated

Reply:
Oh! OK, that makes more sense. 😄 Thank you sir!